.github/workflows/continuous-integration.yml

```diff
@@ -69,9 +69,58 @@ jobs:
       - name: "Run a static analysis with vimeo/psalm"
         run: "vendor/bin/psalm --show-info=false --stats --output-format=github --threads=4"
 
+  phpunit-sqlite:
+    name: "PHPUnit with SQLite"
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
+          - "7.4"
+        deps:
+          - fixed
+        include:
+          - deps: low
+            php-version: "7.3"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          coverage: "pcov"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v2"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        run: "composer install --no-interaction --no-progress --no-suggest"
+        if: ${{ matrix.deps == 'fixed' }}
+
+      - name: "Install lowest possible dependencies with composer"
+        run: "composer update --no-interaction --no-progress --no-suggest --prefer-dist --prefer-lowest"
+        if: ${{ matrix.deps == 'low' }}
+
+      - name: "Run PHPUnit"
+        run: "vendor/bin/phpunit -c ci/github/phpunit.sqlite.xml --coverage-clover=coverage.xml"
+
+      - name: "Upload Code Coverage"
+        uses: "codecov/codecov-action@v1"
+
   phpunit-oci8:
     name: "PHPUnit on OCI8"
     runs-on: "ubuntu-latest"
+    needs: phpunit-sqlite
```

Comment:
That would make `phpunit-sqlite` inconsistently named with other jobs. I will try to create a job named `smoke-check` and make it depend on the `phpunit-sqlite` job. That way, it will no longer look weird, and we can add more checks as part of the smoke check.

Reply:
There does not seem to be such a thing as empty jobs, so I'll just use your suggestion.